data/json/recipes/practice/ranged.json

```diff
@@ -61,6 +61,24 @@
     "components": [ [ [ "arrow_fire_hardened_fletched", 10 ] ] ],
     "byproduct_group": [ { "item": "arrow_fire_hardened_fletched", "charges": [ 0, 2 ] } ]
   },
+  {
+    "id": "prac_crossbow_beg",
+    "type": "practice",
+    "//": "shooting against a hard target should damage arrows",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "crossbows (beginner)",
+    "description": "Practice firing unfletched bolts at a spot on the wall.",
+    "skill_used": "rifle",
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "rifle", 1 ] ],
+    "book_learn": [ [ "recipe_bows", 0 ], [ "manual_archery", 0 ], [ "book_archery", 0 ] ],
+    "tools": [ [ "bullet_crossbow", "crossbow", "compositecrossbow", "compcrossbow", "huge_crossbow" ] ],
```

Comment:
Should put those into a requirement and use that instead I think, that way when more crossbows are added one can just go edit the requirement and all training recipes will update.

Reply:
I've changed it to a requirement entry.